src/components/molecules/LiveSchedule/LiveEvent.tsx

```diff
@@ -1,64 +0,0 @@
-import React from "react";
```

Comment:
What was this file refactored/renamed into, if anything?

Reply:
It was the copy of `EventDisplay` component

Reply:
Cool, thanks.